docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -191,13 +191,13 @@ For the automated certification kit to run successfully, there are currently the
 
 The XenServer certification kit needs to be run with the latest version of the corresponding XenServer Release. Make sure that XenServer 8 has been updated to the latest version before testing. Once the above environment has been set up, please download the xenserver-server-cert-kit-xs8.iso supplemental pack as provided by XenServer, and copy the ISO onto the /root directory of the Dom0 filesystem residing on the pool coordinator host. Use the following command to install server certification kit into all hosts in the pool:
 
-    xe update-upload file-name=”/root/xenserver-server-cert-kit-xs8.iso”   
+    xe update-upload file-name=/root/xenserver-server-cert-kit-xs8.iso 
 
 The command returns the update UUID of server certification kit package on successful upload.
 
     xe update-apply uuid=<update uuid of server certification kit> --multiple  
 
-Upload "vpx-dlvm.xva” file to the server certification kit home folder on all hosts "/opt/xensource/packages/files/auto-cert-kit" .
+Upload "vpx-dlvm.xva” file to the server certification kit home folder on **all hosts** "/opt/xensource/packages/files/auto-cert-kit" .

```

Comment:
where is the ""vpx-dlvm.xva” for uploading? 
I didn't see the steps mentioned this here?

Reply:
How to create "vpx-dlvm.xva" has been described  in the "XenServer Installation" section. So the operator should know where the "XenServer Installation" is.